### PR TITLE
CY-1720 Enabled npm audit in Circle CI build

### PR DESCRIPTION
**Pull request by kubama**
- updated `decompress` and `pm2` to fix vulnerability issues
- added audit job to Circle CI build

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -110,3 +119,9 @@ workflows:
       - checkDocs:
           requires:
             - build
+      - audit:
+          requires:
+            - build
```
Comment (qooban) [inline .circleci/config.yml:124]: Does it require `build`?

File: backend/package.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
   "author": "Cloudify",
   "main": "server.js",
   "scripts": {
+    "audit": "npm audit --production --audit-level=moderate",
```
Comment (qooban) [inline backend/package.json:10]: Why we explicitly set audit-level here?

Comment (kubama) [reply backend/package.json:10]: This sets the minimum vulnerability level on which the build will fail. In this case the build will fail if there is moderate, high or critical vulnerability.
At the moment there are some low vulnerabilities reported which I didn't consider to be something worth addressing. If you think low level should also fail the build I can try to resolve them.

Comment (qooban) [reply backend/package.json:10]: I think that we should work towards having consistent setting. If we are expecting low level vulnerabilities to be reported on front-end part, let's also have the same level set for back-end part. Let's invest some more time and try to resolve low vulnerabilities. If there will be time-consuming blockers we can create separate issue to fix that, but I would like that we will end with consistent setting across UI projects within CY-1720.

File: .circleci/config.yml [hunk 3]
```diff
@@ -95,6 +95,15 @@ jobs:
...
 
+  audit:
+    <<: *default-env
+
+    steps:
+      -   attach_workspace:
+            at: ~/
+      -   run:
+            name: Run npm production dependencies security audit
+            command: npm run audit
```
Comment (qooban) [inline .circleci/config.yml:106]: Probably we can reduce the indent here.

**Pull request merged**